src/platform/forms/save-in-progress/FormStartControls.jsx

```diff
@@ -45,6 +50,7 @@ class FormStartControls extends React.Component {
   };
 
   startOver = () => {
+    this.captureAnalytics();
```

Comment:
Do we have requirements around tracking start and restart events separately?

Reply:
Not that I know of. @djmunoz1 Do you know?